packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/marker/cozy_marker/cozy_marker_data.dart

```diff
@@ -36,14 +37,21 @@ class CozyMarkerData {
   /// Wether marker has a little pointer below it or not.
   final bool hasPointer;
 
+  /// Wether marker has elevation or not.
+  final bool hasElevation;
+
   /// State of the marker
   /// It can have either default or pressed state.
   final CozyMarkerState state;
 
   /// Variant can be default or special. It affects specially the icon of the marker.
   final CozyMarkerVariant variant;
 
-  /// Size can be small or big.
+  /// Size can be:
+  /// - Mini
+  /// - MiniDot
```

Comment:
what is the difference between `mini` and `miniDot`?

Reply:
also `small`, and `big` in this case would be the normal price marker?

Reply:
mini is this elongated width version and miniDot is just the small circle

small and big remain being the price marker, it was a cozy definition

![image](https://github.com/fulpismo/plugins/assets/22605271/5a5bdfdc-fb35-46aa-913d-60460d8432f8)